testutil/ciao-down/prepare.go

```diff
@@ -110,6 +119,26 @@ func hostSupportsNestedKVM() bool {
 	return hostSupportsNestedKVMIntel() || hostSupportsNestedKVMAMD()
 }
 
+func createWorkload(ws *workspace, vmType string) (*workload, error) {
+	localPath := filepath.Join(ws.Home, ".ciao-down", "workloads",
+		fmt.Sprintf("%s.yaml", vmType))
+	ud, err := ioutil.ReadFile(localPath)
+	if err == nil {
+		return &workload{userData: string(ud)}, nil
+	}
+
+	p, err := build.Default.Import(ciaoDownPkg, "", build.FindOnly)
```

Comment:
This use of build.Default.Import() is cool. I think if we ever tried to package Ciao into debs/rpms this code would need to be modified to look into a system location too.